src/SFA.DAS.AssessorService.Application/Handlers/OppFinder/GetOppFinderApprovedStandardDetailsHandler.cs

```diff
@@ -35,11 +35,11 @@ public async Task<GetOppFinderApprovedStandardDetailsResponse> Handle(GetOppFind
 
             string eqaProvider;
 
```

Comment:
Better to use var eqaProviderContactEmail = result.OverviewResult?.EqaProviderContactEmail; instead of duplicating the path in one statement